Framework7/test-to-delete.rb

```diff
@@ -1,7 +1,9 @@
 require_relative '../Framework7/src/helpers/rest_client/api_rest_client'
```

Comment:
this file should not be uploaded since that it is not executed during the cucumber report. it only should be used for your development